db/migrate/20220629173526_create_articles.rb

```diff
@@ -1,12 +1,14 @@
+# frozen_string_literal: true
+
 class CreateArticles < ActiveRecord::Migration[6.0]
   def change
     create_table :articles do |t|
-      t.string :title, null: false
+      t.string :title, null: false, index: true
```

Comment:
Hmmmm, where schema.rb file? Why I don't see it in this pr?

Reply:
Where is the field status??

Reply:
@mmrshk "status" added by another migration